data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -330,6 +420,15 @@ private StructLikeSet rowSetWithoutIds(int... idsToRemove) {
     return set;
   }
 
+  private StructLikeSet rowSetWithoutIds(Set<Integer> idSet, Table iTable, List<Record> recordList) {
```

Comment:
Nit: the name `iTable` is odd. Why not just `table`?

Reply:
I originally used table, but code style check will report hidden field errors.Maybe there is a conflict with a property called table. iTable means iceberg table.